src/App.css

```diff
@@ -1,43 +1,32 @@
 .App {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin: 0 auto;
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.FAQ {
+  background-color: aliceblue;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+h1 {
+  margin: 40px auto;
+  text-align: center;
 }
 
-.App-link {
-  color: #61dafb;
+ul {
+  margin-top: 40px;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-
-  to {
-    transform: rotate(360deg);
-  }
+li {
+  list-style: none;
+  display: inline-flex;
+  width: 120px;
+  padding: 0 30px;
+  color: orange;
 }
 
-button {
-  font-size: calc(10px + 2vmin);
+a:link {
```

Comment:
We should make this `NavBar a:link` to not accidentally remove hyperlink decorations across the website.

Reply:
Made this change - you can see it here:   c163b311df9a0e809aef9ae290708dc32aaf5536

Reply:
Made this change.  You can see it here -  203e2544b5c838199455aa5dad3a55498295eb76

Reply:
Made this change.  You can see it here:  203e2544b5c838199455aa5dad3a55498295eb76